### ISSUE TITLE
Batman Vengeance: Major Stuttering FMV Video Content

### ISSUE THREAD

**Issue by GhostRequiemX1** [xemu-project/xemu#1775]
### Title

https://xemu.app/titles/55530001/#Batman-Vengeance

US-001

### Bug Description

Batman Vengeance cutscenes have these consistent momentary long stutters as it plays through its the videos therefore as a result takes longer to get to gameplay but the gameplay manages to run as normal.  

### Expected Behavior

FMV's should be playing at their native speed

### xemu Version

xemu version: v0.7.132 

### System Information

OS: Windows 10 22H2
CPU: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz   3.70 GHz
GPU: Nvidia GeForce 750 Ti
GPU Driver: 4.0.0 Nvidia 561.09 
GPU Shader: 4.0.0 via Cg complier 

### Additional Context

[xemu.log](https://github.com/user-attachments/files/17067467/xemu.log)

**Comment by Henta1Sn1per** [comment]
Yeah, I tried loading the game up a couple times and thought the entire game was going to stutter. Today I officially let the cutscenes run before exiting it out again, and the gameplay runs pretty smooth! hopefully this gets patched soon.

**Comment by Triticum0** [comment]
Note Using Vulkan seem to alleviate most of the stuttering would recommend using it for this game. If the cutscences seem fixed to you, rename the issue to point to it only being an issue on opengl.

**Comment by Sagadali523** [comment]
Still having the same solution as well

**Comment by audiowylde2345** [comment]
Yeah this is unbearably terrible even though the gameplay is fine It takes forever to get through cutscenes that are minutes long I thought it was maybe resolution settings or some sort of scaling issue but it doesn't matter.
